README.md

```diff
@@ -35,13 +35,24 @@ Multicodec is used in various [Multiformats](https://github.com/multiformats/mul
 
 Find the canonical table of multicodecs at [table.csv](/table.csv). There's also a sortable [viewer](https://ipfs.io/ipfs/QmXec1jjwzxWJoNbxQF5KffL8q6hFXm9QwUGaa3wKGk6dT/#title=Multicodecs&src=https://raw.githubusercontent.com/multiformats/multicodec/master/table.csv).
 
+### Status
+
+Each multicodec is marked with a status:
+
+* draft - this codec has been reserved but may be reassigned if it doesn't gain wide adoption.
+* standard - this codec has been widely adopted and may not reassigned.
```

Comment:
What about renaming "standard" to "permanent" like the [URI Schemes Registry](https://www.iana.org/assignments/uri-schemes/uri-schemes.xhtml) is doing it. That would make clear that having a standard, might still not be enough to get the permanent assignment here.

I think "draft" is a good name.